app/views/posts/_form.html.slim

```diff
@@ -22,7 +22,10 @@
     = form.text_area :description, id: :post_description
   .field
     = form.label :visibility
-    = form.text_area :visibility, id: :post_visibility
+    - if current_user.admin? || current_user.publisher?
```

Comment:
We should avoid some kind of expressions(two expressions concated with ||) in Rails views. 
V  in MVC stands for View.

Secondly, you defined an ability in https://github.com/deHelden/news-website/pull/3/files#diff-e9c04d21b8c4d777e4231c4c6ce5f8cb ?
You should define this logic in ability file, this will be more clear.